src/rl_framework/util/saving_and_loading/connector/hugging_face_connector.py

```diff
@@ -228,14 +222,18 @@ def upload(
                 "tags": [environment_name, "reinforcement-learning", "rl-framework"],
             }
 
+            metrics_value = "not valid"
+            if mean_reward and std_reward and isinstance(mean_reward, float) and isinstance(std_reward, float):
```

Comment:
This one is difficult. We do not know for sure that a `mean_reward` will be passed. `variable_values_to_log` can have 1) another metric passed (e.g., max_reward) or even no metric passed at all. Maybe we should not call `metadata_eval` at all if we are not sure? Or maybe require a second parameter passed called `metric_to_log`?

Reply:
Current solution of reporting "not evaluated" if "mean_reward" is not present is hard-coded but good. Let's keep it.